stackstorm-compose/st2.docker.conf

```diff
@@ -0,0 +1,23 @@
+# /etc/st2/st2.docker.conf
+#
+# This is override st2 config file which will be passed to any st2 command, like this:
```

Comment:
I would say this sentence needs to be rephrased. `This is override st2 config file which...` sounds like confusion or mix of 2 sentences.

Reply:
This file was originally copied from base/files/tmp/st2.docker.conf, but I have rewritten the comment for stackstorm-compose.